index.html

```diff
@@ -99,7 +111,7 @@ <h2 class="project_title">
                         </button>
                      </div>
                  </li>
-                 <li class="project">
+                 <li class="project reverse_row">
```

Comment:
- [ ] Let's kindly align these text blocks like in the Figma:

Your project:

![image](https://user-images.githubusercontent.com/57421823/149130919-923517db-cd0b-40bc-afbf-b10bef2a1ba5.png)

The Figma:

![image](https://user-images.githubusercontent.com/57421823/149130962-597b3ec0-7f08-4950-ab5e-4170725a7a5d.png)

That way, your projects section will be a lot more consistent 💪